<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1365</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BAG FINDER</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(212, 212, 212);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1350</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 248, 248);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font: 11pt &quot;Sans Serif&quot;;
background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-radius: 17px;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table border=&quot;0&quot; style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 75 BOLD 9pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>IATA or PSEUDO ID</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="12">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 75 BOLD 9pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>AREA</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="22">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 75 BOLD 9pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>LOGS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="26">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 75 BOLD 9pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>SEARCH</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_bag_no">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 BOLD 9pt &quot;Sans Serif&quot;;
background-color: rgb(0, 135, 203);
color: rgb(255, 255, 255);
border-radius: 5px;
margin-left:2px;
margin-right:2px;
padding:2px;
margin:1px;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="12">
        <widget class="QComboBox" name="comboBox_area">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 BOLD 9pt &quot;Sans Serif&quot;;
background-color: rgb(0, 135, 203);
border-color: rpg(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 5px;
margin-left:2px;
margin-right:2px;
padding:2px;
margin:1px;</string>
         </property>
         <item>
          <property name="text">
           <string>2-1(WEST)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2-3(Mid-WEST)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2-4(Mid-EAST)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2-6(EAST)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2-1 &amp; 2-3(WEST SIDE)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2-4 &amp; 2-6 (EAST SIDE)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ALL AREA</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="22">
        <widget class="QComboBox" name="comboBox_logs">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 BOLD 9pt &quot;Sans Serif&quot;;
background-color: rgb(0, 135, 203);
border-color: rpg(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 5px;
margin-left:2px;
margin-right:2px;
padding:2px;
margin:1px;</string>
         </property>
         <item>
          <property name="text">
           <string>LAST MODIFIED LOG</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DETAILED LOGS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ALL LOGS</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="26">
        <widget class="QPushButton" name="call_pushButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 BOLD 9pt &quot;Sans Serif&quot;;
background-color: rgb(0, 135, 203);
border-color: rpg(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 5px;
margin-left:2px;
margin-right:2px;
padding:2px;
margin:1px;</string>
         </property>
         <property name="text">
          <string>CALL</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: 75 BOLD 9pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>MV3D</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="8">
        <widget class="QComboBox" name="comboBox_mv3d">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 BOLD 9pt &quot;Sans Serif&quot;;
background-color: rgb(0, 135, 203);
border-color: rpg(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 5px;
margin-left:2px;
margin-right:2px;
padding:2px;
margin:1px;</string>
         </property>
         <item>
          <property name="text">
           <string>3D0037</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0040</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0035</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0036</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0038</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0039</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0025</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0017</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0023</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0022</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0033</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0032</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0020</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0028</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0021</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0026</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0027</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0024</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0044</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0046</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0043</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0045</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0041</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D0042</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="18">
        <widget class="QDateEdit" name="date_selection_box">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 BOLD 9pt &quot;Sans Serif&quot;;
background-color: rgb(0, 135, 203);
border-color: rpg(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 5px;
margin-left:2px;
margin-right:2px;
padding:2px;
margin:1px;</string>
         </property>
         <property name="displayFormat">
          <string>dd/MM/yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="date">
          <date>
           <year>2018</year>
           <month>9</month>
           <day>11</day>
          </date>
         </property>
        </widget>
       </item>
       <item row="1" column="21">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="16" colspan="3">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 75 BOLD 9pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>DATE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="10" rowspan="2">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="14" rowspan="2">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="20" rowspan="2">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="24" rowspan="2">
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="19">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="23">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="25">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="15">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="13">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="11">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="9">
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3" rowspan="2">
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="17">
        <widget class="QCheckBox" name="date_checkBox">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Enable</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QRadioButton" name="mv3d_radioButton">
         <property name="text">
          <string>MV3D</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QRadioButton" name="area_radioButton">
         <property name="text">
          <string>AREA</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <spacer name="horizontalSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="6" rowspan="2">
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <spacer name="horizontalSpacer_12">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1365</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
